docs/embedded/development/content-experiences/search-content.md

```diff
@@ -629,7 +635,210 @@ Content-type: application/json
 }
 ```
 
-## Known Limitation
+## Known Limitations
 
 - Search requests run in the context of the signed-in user. Search results are only scoped to enforce any access control applied to the items by the user. For example, search results will include all container or container content matching the search criteria and accessible by the user regardless of whether the SharePoint Embedded application is authorized to access. You should specify the desired container type by including the ContainerTypeId as part of your **queryString** when searching for containers or container content to ensure search results are properly scoped.
 - For your application to access the containers or container content in search results, it must have access permissions to the corresponding container types.
+
+
+# Enumerate (filter) SharePoint Embedded content
+
+
+Content can also be enumerated using URL parameters to return specific content in SharePoint Embedded containers. This does not use the search API to retrieve items.  See the [enumerate query parameter](https://learn.microsoft.com/en-us/graph/filter-query-parameter?tabs=http) reference.
```

Comment:
Per automated build report findings, invalid link:

- no absolute links - use relative
- no hard-coded locale in URL... remove